src/networking.c

```diff
@@ -1505,6 +1506,20 @@ void clientAcceptHandler(connection *conn) {
         }
     }
 
+    /* Auto-authenticate from cert_user field if set */
+    sds username = connGetPeerUsername(conn);
+    if (username != NULL) {
+        user *u = ACLGetUserByName(username, sdslen(username));
+        if (u) {
+            c->user = u;
+            c->flag.authenticated = true;
+            serverLog(LL_VERBOSE, "TLS: Auto-authenticated client as %s",
+                      server.hide_user_data_from_log ? "*redacted*" : u->name);
+        } else {
+            serverLog(LL_NOTICE, "TLS: Auto-authentication failed, user '%s' not found", username);
```

Comment:
```suggestion
            serverLog(LL_NOTICE, "TLS: Auto-authentication failed, user '%s' not found", username);
```
I wonder if this should go in the ACL log instead? This is basically a failed authentication attempt. That removes the concern about dropping user data to the disk and also includes a lot of useful information like IP.

Reply:
Sounds reasonable. I'm not familiar with the ACL log.

Reply:
So no change need right here ?

Reply:
The suggestion is to log using the ACL log instead of serverLog. It's another way to log things, stored in memory instead of on disk. The username may be considered a little sensitive to store in logs. I guess it's this function: `addACLLogEntry`, https://github.com/valkey-io/valkey/blob/8.1.1/src/acl.c#L2604

It looks like it assumes the client is running a command, so it's accessing `c->argv` and/or `c->cmd`, but if the `object` argument (an sds string) is passed to the function, it uses this instead of taking information from the current command.

Not sure if we should invent some new `reason`. I guess we can. Maybe you can try, or do you @madolson have an idea?

Reply:
My top preference would be to add a new reason, like ACL_INVALID_TLS_CERT_AUTH. That seems very explicit on what happened.

Reply:
Now that ACL log was added, delete the logging using serverLog. We don't want sensitive information in the log and we don't want to flood the log.

```suggestion
```

Reply:
Done removed the `serverLog`